pkg/meta/tkv.go

```diff
@@ -468,6 +490,17 @@ func (m *kvMeta) doInit(format *Format, force bool) error {
 			tx.incrBy(m.counterKey("nextInode"), 2)
 			tx.incrBy(m.counterKey("nextChunk"), 1)
 		}
+
+		// cache all acls
+		acls, err := m.scanValues(m.fmtKey("R"), -1, nil)
```

Comment:
`doInit` is used to create a new volume. Do this in `NewSession`?

Reply:
move acl cache Initialization (engine.cacheACLs) to NewSession